chain-impl-mockchain/src/testing/gen/mod.rs

```diff
@@ -102,4 +105,19 @@ impl TestGen {
             })
             .build()
     }
+
+    pub fn ledger() -> Ledger {
+        let leader_pair = TestGen::leader_pair();
+        let header_id = TestGen::hash();
+        let mut ie = ConfigParams::new();
+        ie.push(ConfigParam::Discrimination(Discrimination::Test));
```

Comment:
those values are not truly random, but we can live with it for the moment. I imagine in the future we will randomize some of the settings like slot_duration or slots_per_epoch.

Reply:
I left a TODO behind for the time being.